src/app.js

```diff
@@ -0,0 +1,33 @@
+#!/usr/bin/env node
+
+const fs = require('fs');
+const { validateLinks } = require('./index')
+
+function findFiles() {
+    let index = process.argv.indexOf('--file');
+    let uri = process.argv[index + 1];
+    if (index < 0 || !uri) return console.log("no hay archivo")
+    readFiles(uri)
+    return uri
```

Comment:
si te refieres al archivo queda mejor el nombre de file
```suggestion
    return md
```